### PR TITLE
Added all exclusive equipment.

### PR DESCRIPTION
**Pull request by Dimbreath**
Added the missing exclusive equipment.

### CODE REVIEW

File: src/ex_equip/prototype-crown.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "heroId": "chloe",
+    "name": "Prototype Crown",
+    "description": "Equipment that contains a set of emergency commands and looks like a tiny crown.",
+    "stat": {
+        "type": "attack",
+        "min": 0.07,
+        "max": 0.14,
```
Comment (AlexeiA) [inline src/ex_equip/prototype-crown.json]: JSON standard doesn't allow trailing comma

Comment (Dimbreath) [reply src/ex_equip/prototype-crown.json]: Oh oops. I didn't notice that. Thanks for letting me know!

File: src/ex_equip/azure-dragons-spirit.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "min": 0.8,
+        "max": 0.16
+    },
+    "enhancements": [
+        {
+            "slot": 1,
+            "description": "35% chance to extend the caster's buff durations by 1 turn when using Double Slash."
+        },
+        {
+            "slot": 2,
```
Comment (RaphaelDDL) [inline src/ex_equip/azure-dragons-spirit.json]: `slot` is the skill slot. For example, Yufine exp equip has two sk1 and one sk3

so the slots had to be 1, 1, 3

File: src/ex_equip/dun-blyraias-dark-blessing.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    "heroId": "wanderer-silk",
+    "name": "Dun Blyraia's Dark Blessing",
+    "description": "A quiver made with Mana from the dimension that Wanderer Silk came from, known to maximize her concentration.",
+    "stat": {
+        "type": "spd",
+        "min": 5,
+        "max": 10
+    },
+    "enhancements": [
```
Comment (RaphaelDDL) [inline src/ex_equip/dun-blyraias-dark-blessing.json]: as above, slots had to be 1,3,3

**Pull request merged**